fragment.go

```diff
@@ -1452,6 +1491,13 @@ func track(start time.Time, message string, stats StatsClient, logger Logger) {
 }
 
 func (f *fragment) snapshot() error {
+	return snapshot(f, f.storage)
+}
+
+// snapshot writes the fragment f with bm as the data. It is unprotected, and
+// f.mu must be locked when calling it.
+func snapshot(f *fragment, bm *roaring.Bitmap) error { // nolint: interfacer
```

Comment:
calling this function `snapshot` is confusing, not only because of it's similarity to `fragment.snapshot`, but because I don't think it's actually doing a snapshot, but rather an unprotected write.